app/views/pages/faqs.html.erb

```diff
@@ -6,6 +6,270 @@
+        </button>
+        <div id="faq-overlimit-content" class="usa-accordion-content">
+        </div>
+      </li>
+
+      <li class="faq-accordion" data-categories="hhg,tsp">
+        <button class="usa-accordion-button faq-accordion-button" aria-controls="faq-overweight-content">
+          My TSP told me my shipment is overweight; what are my options?
+        </button>
+        <div id="faq-overweight-content" class="usa-accordion-content">
+        </div>
+      </li>
+
+      <li class="faq-accordion" data-categories="techhelp">
+        <button class="usa-accordion-button faq-accordion-button" aria-controls="faq-browser-content">
+          What should I do if I can't get DPS to work in my web browser?
+        </button>
+        <div id="faq-browser-content" class="usa-accordion-content">
+        </div>
+      </li>
+
+      <li class="faq-accordion" data-categories="techhelp">
+        <button class="usa-accordion-button faq-accordion-button" aria-controls="faq-eta-content">
+          How do I log into DPS using ETA?
+        </button>
+        <div id="faq-eta-content" class="usa-accordion-content">
+        </div>
+      </li>
+
+      <li class="faq-accordion" data-categories="scheduling">
+        <button class="usa-accordion-button faq-accordion-button" aria-controls="faq-peak-season-content">
+          What is peak season, and when is it?
+        </button>
+        <div id="faq-peak-season-content" class="usa-accordion-content">
+        </div>
+      </li>
+
+    </ul>
+
   </div>
 </div>
+
+<script>
+
+var tagcheckboxes = document.querySelectorAll('#faq-filters input');
+var faqAccordions = document.querySelectorAll('#faq-accordions .faq-accordion');
+var i;
+
+function intersectArrays(arr1, arr2) {
+  var v1, v2;
+  for (var i1 = 0; i1 < arr1.length; i1++) {
+    var v1 = arr1[i1];
+    for (var i2 = 0; i2 < arr2.length; i2++) {
+      var v2 = arr2[i2];
+      if (v1 == v2) {
+        return true;
+      }
+    }
+  }
+  return false;
+}
+
+var checkboxOnClick = function() {
+  var categories = [];
+  for (i = 0; i < tagcheckboxes.length; i++) {
+    var ck = tagcheckboxes[i];
+    if (ck.checked)
+      categories.push(tagcheckboxes[i].value);
+  }
+
+  // if nothing is checked, show all the FAQs
+  if (categories.length == 0) {
+    for (i = 0; i < faqAccordions.length; i++) {
+      faqAccordions[i].style.display = "";
+    }
+    return;
+  }
+
+  for (i = 0; i < faqAccordions.length; i++) {
+    var acc = faqAccordions[i];
+    var tags = acc.getAttribute("data-categories").split(",");
+
+    if (intersectArrays(categories, tags))
+      // a tag matched an enabled category, so make the accordion visible
+      acc.style.display = "";
+    else
+      // no tags matched an enabled category, so hide the accordion
+      acc.style.display = "none";
+  }
+}
+
+for (i = 0; i < tagcheckboxes.length; i++) {
+  tagcheckboxes[i].onclick = checkboxOnClick;
+}
+
+</script>
```

Comment:
+1000 More specifically, I'd like to see this code in an IIFE and in its own `.js` file.